rebar.config

```diff
@@ -14,7 +14,7 @@
   {jsone, "~>1.2.3"},
   fifo_utils,
   {msgpack, "0.3.5"},
-  {dqe, ".*", {git, "https://github.com/dalmatinerdb/dqe.git", {branch, "master"}}},
+  {dqe, "~>0.3.21"},
```

Comment:
The test branch is tagged to gh branches instead of packages to allow iterating changes w/o publishing them. I.e. allowing to change things in dqe, pushing to git but only publishing to hex once the all the changes are stable. I'd like to keep it that way, dev will then be tagged to the corresponding version.